res/figma-fonthelper.service

```diff
@@ -4,7 +4,7 @@ After=systemd-user-sessions.service
 
 [Service]
 Type=simple
-ExecStart=${XDG_CONFIG_HOME}/figma-fonthelper/fonthelper
+ExecStart=/bin/bash -c '${HOME}/.local/share/figma-fonthelper/fonthelper'
```

Comment:
This should also be changed to `${XDG_DATA_HOME:-$HOME/.local/share}` instead of `${HOME}/.local/share`.